src/SayMore/UI/Overview/Statistics/StatisticsViewModel.cs

```diff
@@ -101,20 +102,19 @@ public IEnumerable<ComponentRoleStatistics> GetComponentRoleStatisticsPairs()
 		/// ------------------------------------------------------------------------------------
 		private long GetTotalComponentRoleFileSizes(ComponentRole role)
 		{
-			long bytes = 0;
-			foreach (var info in _backgroundStatisticsGather.GetAllFileData())
-			{
-				if (role.IsMatch(info.MediaFilePath))
-					bytes += info.LengthInBytes;
-			}
-
-			return bytes;
+			// do not include the generated oral annotation file in the calculations
+			return _backgroundStatisticsGather.GetAllFileData()
```

Comment:
Seems that this method and GetRecordingDurations need identical logic to get the file data over which they are to work. Therefore, this should be pulled out into a private helper method they can share, rather than duplicating the logic. Also, for improved efficiency, I think the two Where clauses should be joined into one.

Reply:
Done.